edn/tests/tests.rs

```diff
@@ -33,6 +33,12 @@ fn k_plain(name: &str) -> Value {
     return Keyword(symbols::Keyword::new(name));
 }
 
+// Helper for making wrapped symbols with a namespace
```

Comment:
Hmm.  I feel like there's a `From<(&str, &str)>` for `symbols::NamespacedSymbol` lurking in here, but this'll do for now.